FirebaseMLModelDownloader/Sources/ModelInfoRetriever.swift

```diff
@@ -307,6 +294,7 @@ class ModelInfoRetriever {
                                                                  .statusCode))
               completion(.failure(.invalidArgument))
             case 401, 403:
+              /// Error could be due to FirebaseML API not enabled for project, or invalid permissions.
```

Comment:
Nit: 2 `//` only

Reply:
Updated everywhere not a property/method/class description to be `//`.